documentation/changelog.rst

```diff
@@ -35,6 +38,8 @@ Bugfixes
 v0.4.0 | April 29, 2021
 ===========================
 
+.. warning:: Upgrading to this version requires running ``flexmeasures db upgrade``.
```

Comment:
Shall we add a recommendation to make a backup first using ``flexmeasures db-ops dump``?